src/components/Card/stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import Card from './';
+
+const value = 13;
```

Comment:
I still prefer to caps lock my `const`s, but I'm just chiming in, not concerned.